packages/block-library/src/cover/style.scss

```diff
@@ -50,6 +50,10 @@
 		background-color: $black;
 	}
 
+	[class*="-background-color"] {
+		background-color: var(--wp--user--preset--background-color);
```

Comment:
I don't understand why we do this if the cover block doesn't have any block supports related to this (color, background)?

Reply:
This is just a side effect of the potentially premature optimisation of `.has-background` class, ie. Cover uses the likes of the `has-white-background-color` preset classes, but doesn't assign the corresponding `.has-background` class. Adding this style just avoids a deprecation to add that class, but better options are probably to rethink that optimisation as you note, or to refactor the cover block to use the color supports and add the associated classes.